examples/replay/config-0.25-degree.yaml

```
@@ -211,3 +211,79 @@ FV3Dataset:
         - tmp
         - ugrd
         - vgrd
+
+MOM6Dataset:
+    file_prefixes:
+        - ocn_
+    
+    path_out: "gcs://noaa-ufs-gefsv13replay/ufs-hr1/0.25-degree/03h-freq/zarr/"
+    coords_path_out: "gcs://noaa-ufs-gefsv13replay/ufs-hr1/0.25-degree/coordinates/zarr/"
+    forecast_hours: [0]
+
+    cycles:
+        start: "1994-01-01T00"
+        end: "2023-12-31T06"
+        freq: "6h"
+
+    time:
+        start: "1994-01-01T00"
+        end: "2023-12-31T06"
```

These are offset from the cycles by -6hr. 
```suggestion
        start: "1993-12-31T18"
        end: "2023-12-31T00"
```